### ISSUE TITLE
Subkey install warnings

### ISSUE THREAD

**Issue by NukeManDan** [paritytech/substrate#7945]
I installed as instructed [here](https://substrate.dev/docs/en/knowledgebase/integrate/subkey) and ran into some minor warnings: 
`cargo install --force subkey --git https://github.com/paritytech/substrate --version 2.0.0`
```
.....
  Downloaded 46 crates (9.6 MB) in 6.88s (largest was `prost-build` at 8.2 MB)
warning: profile package spec `cranelift-codegen` in profile `dev` did not match any packages
warning: profile package spec `cranelift-wasm` in profile `dev` did not match any packages
warning: profile package spec `libm` in profile `dev` did not match any packages

	Did you mean `libc`?
warning: profile package spec `librocksdb-sys` in profile `dev` did not match any packages
warning: profile package spec `nalgebra` in profile `dev` did not match any packages
   Compiling libc v0.2.82
.....
```
This is super minor, as the install worked fine.

**Comment by bkchr** [comment]
This is no Substrate specific warning. I assume Cargo prints this because we don't require these dependencies for building subkey. 

TLDR: Won't fix on our side ;)
